templates/citizen_project/contact.html.twig

```diff
@@ -0,0 +1,56 @@
+{% extends 'citizen_project/_layout.html.twig' %}
+
+{% block page_title 'Contacter des acteurs de mon projet citoyen' %}
+
+{% block javascripts %}
+    {% import 'javascript.js.twig' as js %}
+    <script src="{{ asset('ckeditor/ckeditor.js') }}"></script>
+    <script src="{{ asset('ckeditor/lang/fr.js') }}"></script>
+    <script type="text/javascript">
+        Kernel.onLoad(function() {
+            {{ js.ckeditor('message') }}
+        });
+    </script>
+{% endblock %}
+
+{% block citizen_project_content %}
+    <main>
+        <section class="l__wrapper--medium space--30-0">
+            <a href="{{ path('app_citizen_project_list_actors', {slug: citizen_project.slug}) }}" class="back-to-list icon--with-text">
+                {{ include('components/caret--left.html.twig') }}
+                Retour à la liste</a>
+
+            <header class="space--30-0">
+                <h1 class="b__nudge--bottom-small text--medium">
+                    Contacter des acteurs de mon projet citoyen
```

Comment:
it's possible to do `{{ block('page_title') }}`

Reply:
Done.